modules/tile-converter/src/lib/utils/write-queue.ts

```diff
@@ -18,6 +21,13 @@ export default class WriteQueue<T extends WriteQueueItem> extends Queue<T> {
     this.writeConcurrency = writeConcurrency;
   }
 
+  async enqueue(val: T) {
+    super.enqueue(val);
+    if (process.memoryUsage().rss > MEMORY_LIMIT) {
```

Comment:
I would add some comments and possibly links to relevant Node.js docs here.

Reply:
Done